src/targets/browser/index.ejs

```diff
@@ -31,6 +31,7 @@
   data-cozy-tracking="{{.Tracking}}"
   data-cozy-app-editor="{{.AppEditor}}"
   data-cozy-icon-path="{{.IconPath}}"
+  data-cozy="{{.CozyData}}"
```

Comment:
hmmm, since you don't modify the `new CozyClient` does that means that CC will check itself in the DOM to get the data?

Reply:
To myself, yes and no... : there is a loadFromDom() call https://github.com/cozy/cozy-client/blob/69e5a405c7c9d2b01b36ee6c389c48a2a51b063c/packages/cozy-client/src/CozyClient.js#L181 but there is no login if there is no option. So how does it work?

Reply:
Solved later on via CozyClient.fromDOM